src/config.js

```diff
@@ -1619,9 +1619,9 @@ export const CONFIG = {
       platform: 'windows10-64',
       benchmarks: DEFAULT_SUITES,
     },
-    win10Laptops: {
-      label: 'Windows 10 64bit (laptops)',
-      platform: 'windows10-64-ux',
+    win10ref2017: {
+      label: 'Windows 10 64bit (2017 reference laptop)',
+      platform: 'windows10-64-ref-hw-2017',
```

Comment:
Is there a way to provide more than one platform name? This would allow us to show the historic data for this platform, before it was renamed.

Reply:
I filed it as issue #179